python/notebooks/hive/HiveToBigquery_vertex_pipeline_pyspark.ipynb

```diff
@@ -0,0 +1,484 @@
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fb9eff8d-6b63-4e8f-b369-68cbb4ef04ee",
+   "metadata": {},
+   "source": [
+    "#### References\n",
+    "\n",
+    "- [DataprocPySparkBatchOp reference](https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.0/google_cloud_pipeline_components.experimental.dataproc.html)\n",
+    "- [Kubeflow SDK Overview](https://www.kubeflow.org/docs/components/pipelines/sdk/sdk-overview/)\n",
+    "- [Dataproc Serverless in Vertex AI Pipelines tutorial](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/ml_ops/stage3/get_started_with_dataproc_serverless_pipeline_components.ipynb)\n",
+    "- [Build a Vertex AI Pipeline](https://cloud.google.com/vertex-ai/docs/pipelines/build-pipeline)\n",
+    "\n",
+    "This notebook is built to run a Vertex AI User-Managed Notebook using the default Compute Engine Service Account.  \n",
+    "Check the Dataproc Serverless in Vertex AI Pipelines tutorial linked above to learn how to setup a different Service Account.  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0852a764-dad3-4f3b-b0b5-a71825469fd3",
+   "metadata": {},
+   "source": [
+    "#### Permissions\n",
+    "\n",
+    "Make sure that the service account used to run the notebook has the following roles:\n",
+    "\n",
+    "- roles/aiplatform.serviceAgent\n",
+    "- roles/aiplatform.customCodeServiceAgent\n",
+    "- roles/storage.objectCreator\n",
+    "- roles/storage.objectViewer\n",
+    "- roles/dataproc.editor\n",
+    "- roles/dataproc.worker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a2fa95e-b745-4649-8040-af99e7a4013c",
+   "metadata": {},
+   "source": [
+    "#### Step 1:\n",
+    "#### Set Google Cloud properties"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b1a04975-769b-4ce4-8b08-fbcb7b7448f1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# User Configuration\n",
+    "# User Inputs\n",
+    "\n",
+    "get_project_id = ! gcloud config list --format 'value(core.project)' 2>/dev/null\n",
+    "PROJECT_ID = get_project_id[0]\n",
+    "REGION = \"\"  # example\"us-west1\"\n",
+    "GCS_STAGING_LOCATION = \"gs://<bucket_name>\" # example \"gs://bucket_name\"\n",
+    "SUBNET = \"\" # example \"projects/<project-id>/regions/<region-id>/subnetworks/<subnet-name>\" \n",
+    "INPUT_HIVE_DATABASE= \"\"\n",
+    "INPUT_HIVE_TABLES= \"\" # example \"table1,table2,table3...\" or \"*\"\n",
+    "OUTPUT_BIGQUERY_DATASET= \"\"\n",
+    "TEMP_BUCKET= \"<bucket_name>\"\n",
+    "HIVE_OUTPUT_MODE=\"\"\n",
+    "HIVE_METASTORE= \"\" # example \"thrift://hive-cluster-m:9083\"\n",
+    "MAX_PARALLELISM=10 # Overwrite the value if you want to increase number of parallel Dataproc Batch Jobs\n",
+    "\n",
+    "# Run Dataproc Templates from Vertex AI Pipelines\n",
```

Comment:
This comment seems like a notebook Mardown cell and not a code cell.
I believe there was a typo. Line 90 to 94 should be in a new cell on top of the notebook